Apps/WebClient/src/ClientApp/src/store/modules/user/actions.ts

```diff
@@ -306,19 +305,9 @@ export const actions: UserActions = {
                                 resolve();
                             });
                     } else {
-                        if (result.resultError?.statusCode === 429) {
```

Comment:
This code doesn't look right but don't we need 429 logic still?

Reply:
@sslaws  Spoke with @MikeLyttle  about this.  That line of code was erroneous.  The 429 is handled in the catch block on line 314.